### DISCUSSION TITLE
chia farm and fork farm

### DISCUSSION THREAD

**Discussion by west-77** [Chia-Network/chia-blockchain#10101] [Support]
### What happened?

The question is, if I run the forks of chia at the same time while farming on chia, does it cause any harm while farming on chia, so it will cause problems for me to find a block on the cha?

### Version

1.2.11

### What platform are you using?

Windows

### What ui mode are you using?

GUI

**Comment by loppefaaret** [comment]
short answer: it could potentially
the way it works is that every 9 or so seconds, a new round starts, and there is a 1 in 512 chance each of your plot files are eligible for containing proofs. so if your farm is of a certaint size, at some point, both chia and a fork is going to clash, and having lookups at the same time - but depending on your farming setup, the size of the farm, the way the drives are controlled and so on, the chances of them colliding, and causing them to slow down so much that it will hurt your chances of getting a potential winning proof scanned, and create a block, is really a relative point that moves... 
some other factors could potentially be the amount of transactions to be handled on each chain that you are running, and ofcourse, how many chains you are farming at the same time... farming chia and one other fork could probably be done on most farms... but farming chia and 4 other forks, things start to excalate quite quickly... 
personally, i would strive to run a high clocked CPU, so that it gives it the speed to handle the single threaded verifications and other stuff that needs to be handled, as quickly as possible, to eliminate any spikes that might collide from multiple chains at the same time.
